autobump-maven/src/test/java/com/github/autobump/maven/model/MavenDependencyResolverTest.java

```diff
@@ -50,11 +50,16 @@ public void TestSuccesresolveProperties() {
     }
 
     @Test
-    public void TestresolveNullProperty() {
-        assertThrows(DependencyParserException.class, () ->
-                dependencyResolver.resolve(
-                        new Workspace("src/test/resources/project_root_support_properties_nullproperty")
-                ));
+    public void TestresolveUndevinedProperty() {
+        Workspace ws = new Workspace("src/test/resources/project_root_support_properties_undefinedproperty");
+        Set<Dependency> deps = dependencyResolver.resolve(ws);
+        assertEquals(
+                Set.of(Dependency.builder()
```

Comment:
I think the assertion here should be that the set of dependencies is empty. Otherwise, Autobump will start looking for this dependency in the VersionRepository using version: ${org.apache.derby.version}

Unresolved version property -> Autobump ignores the dependency.

Reply:
@the-jos Ik denk dat er hier bedoeld wordt om de versions die niet geresolved worden door bv. een malformed of undefined version niet op te nemen in het resolven van de dependencies. Momenteel worden die dependencies mee ingelezen. 

`    @Test
    public void TestresolveMalformedVarProperty() {
        Workspace ws = new Workspace("src/test/resources/project_root_support_properties_badproperty");
        Set<Dependency> deps = dependencyResolver.resolve(ws);
        assertEquals(
                Set.of(Dependency.builder()
                        .group(TEST_DEPENDENCY_GROUP)
                        .name(TEST_DEPENDENCY_NAME)
                        .version("${org.apache.derby.version")
                        .build()),
                deps);
    }`

Deze test zou dan een lege lijst moeten teruggeven in plaats van de malformed version.
Correct me if I'm wrong @timpeeters